### ISSUE TITLE
turbine_job_script bugs

### ISSUE THREAD

**Issue by boverhof** [CCSI-Toolset/turb_client#17]
# Description
Ran into a few python3 issues when using this script for debugging.

- [ ] turbine_job_script filter length
- [ ] turbine_job_script print maps

**Comment by boverhof** [comment]
# turbine_job_script filter length

**Comment by boverhof** [comment]
# turbine_job_script print maps
```
(ccsi-foqus) C:\Users\Administrator\Desktop\turbine_client_test>python \ProgramData\Anaconda3\envs\ccsi-foqus\Scripts\turbine_job_script -b turbine.cfg
Total JOBS: 1
        CREATE JOBS: 0
        <map object at 0x00000230AA87A948>
        PAUSE JOBS: 0
        <map object at 0x00000230AA810408>
        SUBMIT JOBS: 0
        <map object at 0x00000230AA87A9C8>
        TERMINATE JOBS: 0
        <map object at 0x00000230AA810408>
        SUCCESS JOBS: 0
        <map object at 0x00000230AA87AA88>
        RUNNING JOBS: 0
        <map object at 0x00000230AA810408>
        ERROR JOBS: 1
        <map object at 0x00000230AA87A948>
        SETUP JOBS: 0
        <map object at 0x00000230AA810408>
        CANCEL JOBS: 0
        <map object at 0x00000230AA87AB48>
        WARNING JOBS: 0
        <map object at 0x00000230AA810408>
        EXPIRED JOBS: 0
        <map object at 0x00000230AA87AC08>
```
```

**Comment by lbianchi-lbl** [comment]
This will be fixed by #18.
